inception-ui-core/src/main/java/de/tudarmstadt/ukp/inception/ui/core/dashboard/dashlet/CurrentProjectDashlet.html

```diff
@@ -30,8 +30,8 @@
   </wicket:head>
   <wicket:panel>
     <h1><div wicket:id="name" class="block-ajax-editable-label"/></h1>
-
     <p><wicket:container wicket:id="description"/></p>
+    <div wicket:id="inviteLinkPanel"></div>
```

Comment:
As discussed offline, right now we should make this panel a project settings panel because there we have an extension point we can easily plug the new functionality into. On the dashboard, we'd first have to create extensibility.